app/src/layouts/appLayout/appSidebar/appSidebar.jsx

```diff
@@ -0,0 +1,127 @@
+import LaunchesIcon from './img/launches-icon-inline.svg';
+import FiltersIcon from './img/filters-icon-inline.svg';
+import DebugIcon from './img/debug-icon-inline.svg';
+import ProfileIcon from './img/profile-icon-inline.svg';
+import AdministrateIcon from './img/administrate-icon-inline.svg';
+
+@connect((state) => ({
+  activeProject: activeProjectSelector(state),
+  projectRole: activeProjectRoleSelector(state),
+}))
+@track()
+export class AppSidebar extends Component {
+  static propTypes = {
+    projectRole: PropTypes.string.isRequired,
+    onClickNavBtn: PropTypes.func,
+    activeProject: PropTypes.string.isRequired,
+    tracking: PropTypes.shape({
+      trackEvent: PropTypes.func,
+      getTrackingData: PropTypes.func,
+    }).isRequired,
+  };
+  static defaultProps = {
+    onClickNavBtn: () => {},
+  };
+
+  onClickButton = (eventInfo) => {
+    this.props.onClickNavBtn();
+    this.props.tracking.trackEvent(eventInfo);
+  };
+
+  createTopSidebarItems = () => [
+    {
+      onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_DASHBOARD_BTN),
+      link: { type: PROJECT_DASHBOARD_PAGE, payload: { projectId: this.props.activeProject } },
+      icon: DashboardIcon,
+      message: <FormattedMessage id={'Sidebar.dashboardsBtn'} defaultMessage={'Dashboard'} />,
+    },
+    {
+      onClick: this.props.onClickNavBtn,
+      link: {
+        type: PROJECT_LAUNCHES_PAGE,
+        payload: { projectId: this.props.activeProject, filterId: 'all' },
+      },
+      icon: LaunchesIcon,
+      message: <FormattedMessage id={'Sidebar.launchesBtn'} defaultMessage={'Launches'} />,
+    },
+    {
+      onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_FILTERS_BTN),
+      link: { type: PROJECT_FILTERS_PAGE, payload: { projectId: this.props.activeProject } },
+      icon: FiltersIcon,
+      message: <FormattedMessage id={'Sidebar.filtersBtn'} defaultMessage={'Filters'} />,
+    },
+    {
+      onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_DEBUG_BTN),
+      link: {
+        type: PROJECT_USERDEBUG_PAGE,
+        payload: { projectId: this.props.activeProject, filterId: 'all' },
+      },
+      icon: DebugIcon,
+      message: <FormattedMessage id={'Sidebar.debugBtn'} defaultMessage={'Debug'} />,
+    },
+  ];
+
+  createBottomSidebarItems = () => [
+    {
+      onClick: this.props.onClickNavBtn,
+      link: { type: USER_PROFILE_PAGE },
+      icon: ProfileIcon,
+      message: <FormattedMessage id={'Sidebar.profileBtn'} defaultMessage={'Profile'} />,
+    },
+    {
+      onClick: this.props.onClickNavBtn,
+      link: { type: ADMINISTRATE_PAGE },
+      icon: AdministrateIcon,
+      message: <FormattedMessage id={'Sidebar.administrateBtn'} defaultMessage={'Administrate'} />,
+    },
+  ];
+
+  render() {
+    const { projectRole } = this.props;
+
+    const topSidebarItems = this.createTopSidebarItems();
+    projectRole !== CUSTOMER && topSidebarItems.pop();
```

Comment:
I believe that `if..else` construction is more readable in such cases.

Reply:
Fixed